idr0008-rohn-actinome/screenB/idr0008-screenB-bulkmap-config.yml

```diff
@@ -58,6 +58,12 @@ columns:
       columns:
       - name: siRNA Identifier
         include: yes
+        omitempty: no
+      - name: siRNA Identifier
```

Comment:
that is what simon's example shown. We have to pick up existing header and just put empty value